bazel/ui.bzl

```diff
@@ -76,15 +78,21 @@ def _pl_webpack_library_impl(ctx):
         all_files.append(ctx.info_file)
         all_files.append(ctx.version_file)
 
-    cmd = env_cmds + ui_shared_cmds_start + [
+    cp_cmds = ["cp -aL --parents {} $TMPPATH".format(file.path) for file in all_files]
```

Comment:
do we not need to copy the files from the deps and licenses attributes?

Reply:
those are handled in line 85, 86 since those are outputs and not in the src tree and need to be untar-ed and moved etc.